warp10/src/main/java/io/warp10/continuum/gts/GTSHelper.java

```diff
@@ -2649,39 +2669,36 @@ public static Object parseValue(String valuestr) throws ParseException {
                 }
                 closing++;
               }
-              idx1 = closing;
-              idx2 = closing;
-              offset = 1;
+              idxTokenEnd = closing; // idxTokenEnd points to the character just after ']'
```

Comment:
Except if opening is > 0 in which case there should be a parse error because of a missing ]